pkg/instrumentation/annotation.go

```diff
@@ -30,6 +30,7 @@ const (
 	annotationInjectNodeJSContainersName      = "instrumentation.opentelemetry.io/nodejs-container-names"
 	annotationInjectPython                    = "instrumentation.opentelemetry.io/inject-python"
 	annotationInjectPythonContainersName      = "instrumentation.opentelemetry.io/python-container-names"
+	annotationPythonWheelKind                 = "instrumentation.opentelemetry.io/python-wheel-kind"
```

Comment:
I don't like the name of this annotation. For dotnet, it's called `otel-dotnet-auto-runtime`, which is also not great, but we should aim at some consistency here. That aside, `python-wheel-kind` doesn't say anything useful to the user, in my opinion.

Reply:
Do you mean to use something like `otel-python-distribution` that takes (musl / glibc)? Or want to use `otel-python-auto-runtume`? runtime may make more sense in .net world than in python though

Reply:
I think the name should indicate that what we're really looking for here is the type of libc the application was built against. @pavolloffay @jaronoff97 @iblancasa any thoughts on this?

Reply:
I agree

Reply:
Pushed out a new version using `otel-python-platform` as config name with `glibc` and `musl` as accepted values